applications/FluidDynamicsApplication/custom_utilities/solver_settings.h

```diff
@@ -118,12 +119,18 @@ class SolverSettings
     virtual void SetStrategy(StrategyLabel const& rStrategyLabel,
                              typename TLinearSolver::Pointer pLinearSolver,
                              const double Tolerance,
-                             const unsigned int MaxIter) = 0;
+                             const unsigned int MaxIter) {}
+
+    virtual void SetStrategy(StrategyLabel const& rStrategyLabel,
+                             typename TLinearSolver::Pointer pLinearSolver,
+                             const double Tolerance,
+                             const std::size_t MaxIter,
+                             typename TBuilderAndSolverType::Pointer pNewBuilderAndSolver){}
```

Comment:
See comment for the other `SetStrategy`. I would suggest a at least a runtime error here too.